controls/def.cf

```diff
@@ -315,6 +333,10 @@ bundle common def
       ifvarclass => "$(augments_classes_data_keys)";
 
       "$(const.t) $(defvars) = $($(defvars))";
+      "DEBUG $(this.bundle): Agent parsed augments_file"
+        ifvarclass => "have_augments_file.feature_augments";
+      "DEBUG $(this.bundle): Policy parsed augments_file"
```

Comment:
I think this is enough, it would take a fair amount of policy to do all the exact reports about each variable skipping definition because it was already defined and had an augments_file tag.